# Equivalent to make maliput::api::RoadNetwork::road_geometry()->FindRoadPositions(inertial_position, radius).
InertialPosition inertial_position
float64 radius
---
RoadPositionResult[] road_position_results
